lib/puppet/pops/types/p_sem_ver_range_type.rb

```diff
@@ -5,6 +5,10 @@ module Types
 #
 # @api public
 class PSemVerRangeType < PScalarType
+  def self.register_ptype(loader, ir)
+    create_ptype(loader, ir, 'ScalarType')
```

Comment:
ok not important - it was just that the same string is created multiple times = more garbage